cinema_project/wsgi.py

```diff
@@ -0,0 +1,16 @@
+"""
+WSGI config for cinema_project1 project.
```

Comment:
The project name in the comment seems to be incorrect. It mentions 'cinema_project1' instead of 'cinema_project'. Please correct it to avoid confusion.